mineral-ui/mineral-ui
# Improve Icon a11y by using role="img"

### Description

* Updates the Icon component to use role="img" instead of role="icon"
* Updates other component contextual icon CSS selectors

### Motivation and context

closes #463 

### Screenshots, videos, or demo, if appropriate

https://463-icon-role--mineral-ui.netlify.com/

### How to test

1. Run a a11y check on the Icon demo page using chrome devTools and note that there are no longer any warnings about invalid role attribute on the Icon svg's
2. Add a title and a tabIndex to an Icon component in the demo and verify that it is announced properly, and better than it was previously.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

Fixes invalid role attribute

### Checklist
